app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/JenkinsModal.tsx

```diff
@@ -50,52 +49,59 @@ const JenkinsModal: React.FC<ModalProps> = ({
 	isLoading,
 	isCopiedToClipboard
 }: ModalProps): JSX.Element => {
+	console.log('hello?? ', dataTestId);
+	const loadingButtonWidthClassName = dataTestId === DELETE_MODAL_TEST_ID ? deleteLoadingIcon : loadingIcon;
+
 	return (
 		<ModalTransition>
-			{show && (
```

Comment:
Had to pull the rendering logic modal up. Now that we've got the modal potentially rendering if 3 separate spots on the server management page, wires were getting crossed and the modals were playing funny buggers.